### DISCUSSION TITLE
blog/2020/12/01/ipw-binary-continuous/

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#80] [Blog comments]
# blog/2020/12/01/ipw-binary-continuous/

Use R to close backdoor confounding by generating and using inverse probability weights for both binary and continuous treatments

https://www.andrewheiss.com/blog/2020/12/01/ipw-binary-continuous/

**Comment by xinzhuohkust** [comment]
really learned a lot, thank you for your excellent content!
